.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    # Run daily at 430 am PT
+    - cron: '30 11 * * *'
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          stale-issue-message: 'This issue is being marked stale because it has not had any activity for 30 days. Reply below within 7 days if it should be left open.'
```

Comment:
How about a "reply if your issue still isn't solved"?

Reply:
tweaked phrasing:
`This issue is being marked stale because it has not had any activity for 30 days. Reply below within 7 days if your issue still isn't solved, and it will be left open. Otherwise, the issue will be closed automatically.`